src/platform/graphics/webgl/webgl-graphics-device.js

```diff
@@ -192,10 +192,7 @@ function testTextureFloatHighPrecision(device) {
  */
 class WebglGraphicsDevice extends GraphicsDevice {
     /**
-     * The WebGL context managed by the graphics device. The type could also technically be
-     * `WebGLRenderingContext` if WebGL 2.0 is not available. But in order for IntelliSense to be
-     * able to function for all WebGL calls in the codebase, we specify `WebGL2RenderingContext`
-     * here instead.
+     * The WebGL2 context managed by the graphics device.
```

Comment:
This should really say `Create a new WebglGraphicsDevice instance.` And the class description should be update to cover what the `WebglGraphicsDevice` is about (including talking about the fact that it's a WebGL 2.0-based device with no support for WebGL 1.0.

Reply:
This is not a doc for the constructor, but just a `gl` member. The rest is as you described already.

Reply:
Oh, yeah, not sure why I misread that. Anyway, this class should have its own class description instead of cloning the `GraphicsDevice` one. That comment still stands.